Library/Homebrew/utils.rb

```diff
@@ -267,7 +292,11 @@ def curl(*args)
   safe_system curl, *args
 end
 
-def puts_columns(items, highlight = [])
+def output_length string
+  string.gsub(/\033\[(\d;)?\d+m/, "").length
```

Comment:
I think we should make the regexp `/\033\[\d+(;\d+)*m/`. This will handle all cases I can come up with correctly, even those we do not use that currently, e.g. `"\033[1;37;41m"` (bold, foreground color, background color) or `"\e[32;1m"` (foreground color and bold swapped, still valid).